report/src/test/kotlin/com/hartwig/actin/report/interpretation/MolecularDriverEntryFactoryTest.kt

```diff
@@ -178,22 +194,31 @@ class MolecularDriverEntryFactoryTest {
         assertVariantType(bothCavBi.copy(proteinEffect = ProteinEffect.LOSS_OF_FUNCTION), "Mutation (Cancer-associated variant, biallelic)")
         assertVariantType(bothCav.copy(proteinEffect = ProteinEffect.GAIN_OF_FUNCTION), "Mutation (Cancer-associated variant)")
         assertVariantType(bothCavBi.copy(proteinEffect = ProteinEffect.GAIN_OF_FUNCTION), "Mutation (Cancer-associated variant, biallelic)")
-        assertVariantType(bothNoCav.copy(proteinEffect = ProteinEffect.UNKNOWN), "Mutation (No known cancer-associated variant)")
+        assertVariantType(
+            bothNoCav.copy(proteinEffect = ProteinEffect.UNKNOWN),
+            "Mutation (No known cancer-associated variant, high driver)"
+        )
         assertVariantType(
             bothNoCavBi.copy(proteinEffect = ProteinEffect.UNKNOWN),
-            "Mutation (No known cancer-associated variant, biallelic)"
+            "Mutation (No known cancer-associated variant, biallelic, high driver)"
+        )
+        assertVariantType(bothNoCav.copy(proteinEffect = ProteinEffect.NO_EFFECT), "Mutation (No protein effect, high driver)")
+        assertVariantType(bothNoCav.copy(proteinEffect = ProteinEffect.NO_EFFECT_PREDICTED), "Mutation (No protein effect, high driver)")
+        assertVariantType(
+            bothNoCav.copy(proteinEffect = ProteinEffect.LOSS_OF_FUNCTION),
+            "Mutation (No known cancer-associated variant, high driver)"
```

Comment:
Same here, looks weird... loss of function yet "no known cancer-associated variant"?